learning/tour-of-beam/frontend/lib/constants/assets.dart

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+String _getPath(String fileName, String format) {
+  return '$format/$fileName.$format';
+}
+
+final kBeamLogoAsset = _getPath('beam_logo', 'png');
+final kThemeModeAsset = _getPath('theme_mode', 'svg');
```

Comment:
Should we make format constants?

Reply:
I don't like global constants and normally gather them in classes like

```dart
class Assets {
  final static logo = ...
  final static themeMode = ...
}
```

`Assets.logo` is way cleaner.
Same goes for every constant in every file with constants.